### DISCUSSION TITLE
Inverted 3x5 Angle Brackets

### DISCUSSION THREAD

**Discussion by mostlydave** [aderusha/DDD-Printable-Wall-Control-System#46] [Ideas]
The largest Gridfinity centerpiece I can print on my ender 3 pro is 5x7 Gridfinity Frame 3x4.stl. Would it be possible to get an inverted version of the 3x5 Angle Brackets to go with it?

**Comment by aderusha** [comment]
It sure would be possible!  I've added the 3x5 inverted brackets under [Sidepieces/Angle_Brackets](/aderusha/DDD-Printable-Wall-Control-System/tree/main/Sidepieces/Angle_brackets).

**Comment by mostlydave** [comment]
Thanks that was fast!!!
